pkg/loghttp/push/push_test.go

```diff
@@ -80,20 +101,55 @@ func TestParseRequest(t *testing.T) {
 			contentEncoding: `gzip`,
 			valid:           true,
 		},
+		{
+			path:            `/loki/api/v1/push`,
+			body:            deflateString(`{"streams": [{ "stream": { "foo": "bar2" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}`),
+			contentType:     `application/json; charset=utf-8`,
+			contentEncoding: `deflate`,
+			valid:           true,
+		},
 		{
 			path:            `/loki/api/v1/push`,
 			body:            gzipString(`{"streams": [{ "stream": { "foo": "bar2" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}`),
 			contentType:     `application/jsonn; charset=utf-8`,
 			contentEncoding: `gzip`,
 			valid:           false,
 		},
+		{
+			path:            `/loki/api/v1/push`,
+			body:            deflateString(`{"streams": [{ "stream": { "foo": "bar2" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}`),
+			contentType:     `application/jsonn; charset=utf-8`,
+			contentEncoding: `deflate`,
+			valid:           false,
+		},
 		{
 			path:            `/loki/api/v1/push`,
 			body:            gzipString(`{"streams": [{ "stream": { "foo4": "bar2" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}`),
 			contentType:     `application/json; charsetutf-8`,
 			contentEncoding: `gzip`,
 			valid:           false,
 		},
+		{
+			path:            `/loki/api/v1/push`,
+			body:            deflateString(`{"streams": [{ "stream": { "foo4": "bar2" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}`),
+			contentType:     `application/json; charsetutf-8`,
+			contentEncoding: `deflate`,
+			valid:           false,
+		},
+		{
+			path:            `/loki/api/v1/push`,
+			body:            deflateString(`{"streams": [{ "stream": { "foo": "bar2" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}`),
+			contentType:     `application/jsonn; charset=utf-8`,
+			contentEncoding: `deflate`,
+			valid:           false,
+		},
+		{
+			path:            `/loki/api/v1/push`,
+			body:            deflateString(`{"streams": [{ "stream": { "foo4": "bar2" }, "values": [ [ "1570818238000000000", "fizzbuzz" ] ] }]}`),
+			contentType:     `application/json; charsetutf-8`,
+			contentEncoding: `deflate`,
+			valid:           false,
+		},
```

Comment:
These two tests seem to be already added above. Let us remove the duplicate tests.

Reply:
ok